data/assets/examples/renderable/renderabledistancelabel/renderabledistancelabel_example.asset

```diff
@@ -0,0 +1,47 @@
+local moon = asset.require("scene/solarsystem/planets/earth/moon/moon")
+local earth = asset.require("scene/solarsystem/planets/earth/earth")
+
+
+local line = {
+  Identifier = "RenderableNodeLine_example",
```

Comment:
```suggestion
  Identifier = "RenderableDistanceLabel_Example_NodeLine",
```

How about this?  Otherwise we are going to have a naming conflict with the RenderableNodeLine example we are going to write